src/component/variantPage/BasicInfo.tsx

```diff
@@ -507,6 +495,16 @@ export default class BasicInfo extends React.Component<IBasicInfoProps> {
         );
     }
 
+    public filterPillsByKey(keys: string[], renderData: BasicInfoData[]) {
+        return renderData.filter((element) => keys.includes(element.key));
+    }
+
+    public getPillList(keys: string[], renderData: BasicInfoData[]) {
+        return _.map(this.filterPillsByKey(keys, renderData), (data) => {
+            return this.generatePills(data.value, data.key, data.category);
+        });
```

Comment:
I find it more readable this way. Not a big deal though.

```suggestion
        return this
            .filterPillsByKey(keys, renderData)
            .map(data => this.generatePills(data.value, data.key, data.category));
```